pkg/scalers/etcd_scaler.go

```diff
@@ -0,0 +1,299 @@
+package scalers
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
```

Comment:
Please move this down below the commong imports.

Reply:
done

> Please move this down below the common imports.

done!

Reply:
> Please move this down below the common imports.

done!

Reply:
Long time no see. I've finished the e2e test, is there anything else I can do?